apps/protocol-frontend/src/hooks/useDaoUserCreate.ts

```diff
@@ -17,7 +17,7 @@ export const useDaoUserCreate = () => {
       const mutationData = await govrn.guild.user.create({
         data: {
           userId: newDaoUser.userId,
-          guildId: newDaoUser.guildId,
+          guildId: newDaoUser.guildId ?? 0,
```

Comment:
This is a good point -- while I think it's more concise than explicitly setting it, I had added the `guildId:0` initially to be explicit about that code path in the resolver to prevent unintentional DAO creation.

Reply:
guild id is optional actually, but removing this throws error states that guild id is required.
0 is not valid id for any record & it's not used to create a new DAO, only used for updating. 

Another better suggestion would be using update custom resolver instead ot update members, then we don't need a guild id at all.

Reply:
Oh, sorry for not being clear -- it is optional but the permissions change requires us to pass something in, so setting it explicitly to 0 allowed for the code paths in the resolver.